src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewController.swift

```diff
@@ -382,3 +409,51 @@ private extension UITableViewCell {
 		}
 	}
 }
+
+/// Stuff to draw a background to the cell, depending on the position of the cell in the section it gets rounded corners.
+
+private extension UITableViewCell {
+	enum BackgroundLocation: Int {
+		case top = 200_001
```

Comment:
In first sight I was wondering about the magic numbers here. Then I saw, that they have a dependency to other magic numbers (seperators). Would it make sense to place them together at a central point?

Reply:
Hmm i was thinking about this, too, but decided to put these view tags there. Now we have two extension (separators and background) and they capsulate their view tags and the logic inside. If we put the tags together at one place, one could be confused why they are not aligned with the logic.... Do you agree or disagree?

Reply:
Its potentially dangerous. Some future developer could just add a case like `topleft`. And for whatever reasons, think its a good idea to name it "100_001". And then there will be a very weird bug, hard to track down.

Reply:
That's a point. I extracted both view tags (background and separators) to a common central place. (see file  UITableViewCell+ViewTags)